packages/file-upload/src/from-url/styles.ts

```diff
@@ -0,0 +1,42 @@
+import { Box } from '@mui/material';
+import { styled } from '@novatics/styles';
+import { FromUrlProps } from '.';
+
+interface FromURLContainerProps extends Pick<FromUrlProps, 'variant'> {
+  componentDisabled?: boolean;
+}
+
+export const FromURLContainer = styled(Box)<FromURLContainerProps>`
```

Comment:
I find easier to read this styled component when we use something like this(something like it)

```
styled(Box)<FromURLContainerProps>((props) => { 
  const border = props.variant ==  'single'  ? '1px solid' : 'none';

  return {
   border
  }
)
```